fabric/state_continuity_core.py

```
@@ -211,168 +203,171 @@
+                    "analyze_text", {"text": context["text"]}
                 )
-                enriched['mcp_analysis'] = result
+                enriched["mcp_analysis"] = result
         except Exception as e:
             print(f"MCP enrichment failed: {e}")
-        
+
         return enriched
-    
+
     def _apply_privacy_filters(self, context: Dict[str, Any]) -> Dict[str, Any]:
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)